.settings/com.google.gdt.eclipse.core.template.prefs

```diff
@@ -0,0 +1,4 @@
+eclipse.preferences.version=1
+jarsExcludedFromWebInfLib=*/src/test/resources/lib/appengine/appengine-api-labs.jar|*/src/test/resources/lib/appengine/appengine-api-stubs.jar|*/src/test/resources/lib/appengine/appengine-api.jar|*/src/test/resources/lib/appengine/appengine-remote-api.jar|*/src/test/resources/lib/appengine/appengine-testing.jar|*/src/test/resources/lib/httpunit/httpunit.jar|*/src/test/resources/lib/javamail/mail.jar|*/src/test/resources/lib/selenium/selenium-server-standalone-2.46.0.jar
+warSrcDir=src/main/webapp
+warSrcDirIsOutput=true
```

Comment:
Alternatively, I can just overwrite the old `.prefs` file. If the directory is wrong there's no other side effect (as far as I know) other than that the desired effect will not be achieved.
Relative paths for this has been a longstanding issue as seen [here](https://github.com/pruebasetichat/google-plugin-for-eclipse/issues/81), with no resolution yet.